.gitignore

```diff
@@ -29,3 +29,6 @@ cache
 .idea/
 *.pem
 .history
+
+# connectors
+/packages/core/connectors
```

Comment:
also, this will be removed as long as connectors are moved out of this repo right?

Reply:
NO, this is for the downloaded connectors, not the source code.